app/src/main/kotlin/app/futured/androidprojecttemplate/injection/modules/ApplicationModule.kt

```diff
@@ -23,8 +26,14 @@ class ApplicationModule {
 
     @Singleton
     @Provides
-    fun moshi(): Moshi = Moshi.Builder()
-        .build()
+    fun json(): Json = Json(from = Json.Default) {
+        encodeDefaults = true
+        ignoreUnknownKeys = true
+        isLenient = true
+        serializersModule = SerializersModule {
+            contextual(ZonedDateTime::class, ZonedDateTimeSerializer)
```

Comment:
I'm not sure if we want to have this in the template but I would say that it is handy since it is used in almost every project.